Documentation/contributing/testing/ci.rst

```diff
@@ -176,80 +178,70 @@ We are currently testing following kernel - k8s version pairs in our CI:
+
++------------------+--------------------------+
+| PR target branch | Trigger required PR jobs |
++==================+==========================+
+| master           | test-me-please           |
++------------------+--------------------------+
+| v1.9             | test-backport-1.9        |
++------------------+--------------------------+
+| v1.8             | test-backport-1.8        |
++------------------+--------------------------+
+| v1.7             | test-backport-1.7        |
++------------------+--------------------------+
+
+On top of these shorthands for triggering required jobs, one may use any
+of these additional triggers to either trigger optional jobs, or manually
+re-trigger a single job that previously failed (in case of a flake):
+
++--------------------+---------------------------------------------------------------------------+
+| Trigger phrase     | PR job(s) triggered                                                       |
++====================+===========================================================================+
+| test-X.XX-4.9      | Specific K8s version ``X.XX`` on Kernel 4.9                               |
++--------------------+---------------------------------------------------------------------------+
+| test-X.XX-4.19     | Specific K8s version ``X.XX`` on Kernel 4.19                              |
++--------------------+---------------------------------------------------------------------------+
+| test-X.XX-net-next | Specific K8s version ``X.XX`` on Kernel net-next                          |
++--------------------+---------------------------------------------------------------------------+
+| test-gke           | GKE test (runs on K8s version currently marked as "stable" in GKE)        |
++--------------------+---------------------------------------------------------------------------+
+| test-runtime       | Runtime test                                                              |
++--------------------+---------------------------------------------------------------------------+
+| test-upstream      | Upstream test (note: K8s and Kernel versions vary depending on PR branch) |
++--------------------+---------------------------------------------------------------------------+
+| test-4.9           | Latest K8s version on Kernel 4.9                                          |
++--------------------+---------------------------------------------------------------------------+
+| test-4.19          | Latest K8s version on Kernel 4.19                                         |
++--------------------+---------------------------------------------------------------------------+
+| test-net-next      | Latest K8s version on Kernel net-next                                     |
++--------------------+---------------------------------------------------------------------------+
+| test-older-k8s     | All non-required K8s versions on Kernel 4.9 **for ``master`` PRs**        |
```

Comment:
I was thinking last night maybe `test-older-k8s` is actually a bad name as it does not convey the intent truthfully (the oldest k8s version is already tested via `test-me-please`). `test-missed-k8s` was indeed more appropriate. We could actually still use it, instead of deprecating/removing it. Other proposals I had are `test-other-k8s` and `test-optional-k8s`. I think the latter is actually relevant considering the constrast w.r.t. "required" jobs.


What do you think?

Reply:
Any downside to keeping `test-missed-k8s`? That's what everyone is used to.

Reply:
None, except that people might be confused if they try to use on backport PRs (they already should not, but one never knows).

Reply:
Now that all k8s-all are split, it seems unlikely that someone would want to restart them all at once (more likely they'll want to restart one of those jobs or everything with `test-backport-1.X`). If that ever happens, it will just trigger a couple unnecessary jobs, right? If that's indeed the case, I'd say go for it.

Reply:
Changed to `test-missed-k8s`.